setup.py

```diff
@@ -38,7 +38,7 @@
     package_dir={'': 'src'},
     python_requires='>=3.6',
     install_requires=[
-        'pydicom>=1.4,!=2.1.0,!=2.1.1',
+        'pydicom>2.1',
```

Comment:
not sure this is still correct, following discussions?

Reply:
Addressed via 0580ea90d03ec8985f9714f14b1b12d80fb10e00

Reply:
I initially tried setting `>=2.2`, but that currently doesn't work yet. We'll have to update once the 2.2 version is released.